rebalance/rebalance.py

```diff
@@ -585,7 +598,7 @@ def maybe_rebalance_pairs(ch1, ch2, failed_channels: list):
             return result
         amount = min(amount, get_max_amount(i, plugin))
         maxfee = get_max_fee(amount)
-        plugin.log(f"Try to rebalance: {scid1} -> {scid2}; amount={amount}; maxfee={maxfee}")
+        plugin.log(f"Thread{get_thread_id_str()} tries to rebalance: {scid1} -> {scid2}; amount={amount.to_satoshi_str()}; maxfee={maxfee.to_satoshi_str()}")
```

Comment:
you also need to fix the testcase `rebalance/test_rebalance.py::test_rebalance_all` for this, as this is still currently waiting just for "Try to rebalance: ..."

Reply:
The last time I tried (it was more than a year ago) it was so difficult to make the tests work. But OK, I will try.